README.md

```diff
@@ -7,133 +7,49 @@ The first "data-first" and "data-last" utility library designed especially for T
 [![NPM](https://img.shields.io/npm/v/remeda)](https://www.npmjs.org/package/remeda)
 ![Dependencies](https://img.shields.io/librariesio/release/npm/remeda)
 
-## Installation
+## Documentation
 
-```bash
-npm i remeda@beta
-yarn add remeda@beta
-```
+Full documentation on [remedajs.com/docs](https://remedajs.com/docs).
 
-Then in .js or .ts
+For Lodash and Ramda users, check the function mapping on [remedajs.com/mapping](https://remedajs.com/mapping).
```

Comment:
I got used to the chatgpt style, I think longer sentences are better. Something along the lines of: "If you are migrating from Lodash or Ramda, check out our mapping to help you convert your project.

Reply:
i personally prefer brevity over longer sentences, but in this case i think something like

```md
Migrating from Lodash or Ramda? Check the function mapping on [remedajs.com/mapping](https://remedajs.com/mapping).
```

is better